docs/src/domains/steps/tls.md

```diff
@@ -4,22 +4,20 @@ weight: 3
 sidebarTitle: "Custom TLS certificates"
 ---
 
-Platform.sh automatically provides all environments with standard Transport Layer Security (TLS) certificates issued by [Let's Encrypt](https://letsencrypt.org/).
-No further action is required to use TLS-encrypted connections beyond [specifying HTTPS routes](../../define-routes/https.md).
-
-You can also provide your own third-party TLS certificate from the issuer of your choice.
-Platform.sh doesn't charge for using a third-party TLS certificate, although the issuer may.
-Consult your TLS issuer for instructions on how to generate an TLS certificate.
-
-You can use many kinds of certificates, including domain-validated, extended validation, high-assurance, and wildcard certificates.
+{{% tls-introduction %}}
 
 A custom certificate isn't necessary for development environments.
 Platform.sh automatically provides wildcard certificates that cover all `*.platform.sh` domains, including development environments.
 
-If you are using a third-party certificate, seven days before it expires
-Platform.sh issues a Let's Encrypt certificate and replaces the custom certificate with it to avoid interruption in service.
-If you wish to continue using the custom certificate,
-replace it with an updated certificate more than seven days before it expires.
+Platform.sh doesn't charge for using a third-party TLS certificate.
```

Comment:
:pick: The abbreviation is defined in the shortcode but not recognized by the linting.